package/features/viz.feature

```diff
@@ -5,13 +5,6 @@ Feature: Viz plugin in new project
     Background:
         Given I have prepared a config file with example code
 
-    Scenario: Execute viz with Kedro 0.16.1
-        Given I have installed kedro version "0.16.1"
-        And I have run a non-interactive kedro new
-        And I have installed the project's requirements
-        When I execute the kedro viz command "viz"
-        Then kedro-viz should start successfully
-
```

Comment:
While I notice that we did mentioned we are not supporting Kedro 0.16 in the description, that said I wonder what is the reason behind this - is this due to any technical decision ( i.e failed tests, etc)? 

from last I recall [we are still supporting >= Kedro 0.16.6 ](https://github.com/kedro-org/kedro-viz/blob/main/CONTRIBUTING.md#launch-a-development-server-with-a-real-kedro-project), my only worry is that, if we are stopping support for 0.16, then those users on a slightly older version will not get any bug fixes and updates to the flowchart features. 

Not that I am against dropping support with 0.16 at all, perhaps this might be worth a wider discussion with the team so that we are decide and make any needed updates to our docs / announcement to user groups, etc?

Reply:
yes, this was discussed in the backlog grooming session last week. Update on ticket #805